Detectors/TRD/simulation/src/trap2raw.cxx

```diff
@@ -89,12 +89,14 @@ int main(int argc, char** argv)
   }
   o2::conf::ConfigurableParam::updateFromString(vm["configKeyValues"].as<std::string>());
 
-  trap2raw(vm["input-file-digits"].as<std::string>(), vm["input-file-tracklets"].as<std::string>(), vm["output-dir"].as<std::string>(), vm["verbosity"].as<int>(), vm["file-per"].as<std::string>(), vm["rdh-version"].as<uint32_t>(), vm["no-empty-hbf"].as<bool>(), vm["tracklethcheader"].as<int>());
+  const auto& hbfu = o2::raw::HBFUtils::Instance(); // we need the creation time for the link mapping from CCDB
```

Comment:
Hi @shahor02 do you know how I can correctly get the `startTime` from `o2simdigitizerworkflow_configuration.ini`? I tried here simply with `o2::raw::HBFUtils::Instance().startTime` but that gives me 1547590800000 while in the ini file I have `startTime=1662576045021`.
A few lines above I thought the HBFUtils parameters should be updated from the file
```
  std::string confDig = vm["hbfutils-config"].as<std::string>();
  if (!confDig.empty() && confDig != "none") {
    o2::conf::ConfigurableParam::updateFromFile(confDig, "HBFUtils");
  }
```

Reply:
Ah, sorry for the noise I think everything is fine. I was using the default parameter by accident